docs/source/functional.rst

```diff
@@ -124,10 +124,10 @@ vad
 
 .. autofunction:: spectrogram
 
-:hidden:`amplitude_to_DB`
-~~~~~~~~~~~~~~~~~~~~~~~~~
+:hidden:`spectrogram`
+~~~~~~~~~~~~~~~~~~~~~
 
-.. autofunction:: amplitude_to_DB
+.. autofunction:: spectrogram
```

Comment:
Looks like these changes are accidentally introduced.

Could you put `DB_to_amplitude` next to `amplitude_to_DB` and put `griffinlim` next to `spectrogram`, so that (loosely) related functions are in nearby?